cypress/integration/Profile Image Upload Tests/profileImageUpload.specs.js

```diff
@@ -0,0 +1,65 @@
+ /// <reference types="cypress" />
+ import {profileTab} from '../../utils/profileTab'
+
+ describe("NeetoAuth Profile Image upload functionality", () => {
+   beforeEach(() => {
+    profileTab();
+  });
+    
+  it("Uploading image less than 5MB", () => {
+      cy.uploadImg('lessThan5MB.jpg','Profile image successfully updated!');
+  });
+
+  it("Uploading image more than 5MB", () => {
+      cy.uploadImg('moreThan5MB.jpg','Something went wrong.');
+  });
+
+    it("Uploading unallowed file type (.pdf)", () => {
+      cy.uploadImg('wrongFileFormat.pdf','Something went wrong.');
+  });
+
+  it("Uploading multiple images file", () => {
+      cy.get('[data-cy="profile-image-upload-file-field"]').attachFile('./lessThan5MB').attachFile('./lessThan5MB');
+      cy.get('[data-cy=toastr-message-container]').eq(0).should('have.text','Something went wrong.')
+      cy.get('[data-cy=toastr-message-container]').eq(1).should('have.text','Something went wrong.')
```

Comment:
We can avoid these types of selectors, we may face DOM detached issues for it. If we observe, there are two toast messages but the selectors are same. We can use a better logic to assert each toast message.
```suggestion
      cy.get('[data-cy=toastr-message-container]').should('have.text','Something went wrong.')
      cy.get('[data-cy=toastr-message-container]').should('have.text','Something went wrong.')
```